src/libutil/rust-ffi.hh

```diff
@@ -171,7 +197,7 @@ struct Result
     }
 
     /* Rethrow the wrapped exception or return the wrapped value. */
-    T unwrap()
+    T unwrap() &&
```

Comment:
what does `&&` do here?

Reply:
From earlier PR too, it makes something like taking `this` by rvalue reference

Reply:
The point is that since we are moving `data` out, the original is consumed/invalidated. Making it take it by rvalue-refrence means the original can't be used anyways (it was an rvalue or `std::move`d), which implements this idea and roughly matches the Rust semantics.